pages/common/trans.md

```diff
@@ -0,0 +1,23 @@
+# trans
+
+> **Translate Shell** is a command-line translator.
```

Comment:
Why the ** ? The clients do not parse and display according to exact markdown rules. So the ** will be shown as ** itself, not bold text.

Reply:
I just wanted to keep the description like on this page: https://www.soimort.org/translate-shell/
But didn't notice that this will not work. Will remove it